chapter-11/recipe-03/fortran-example/CMakeLists.txt

```diff
@@ -26,10 +26,10 @@ find_package(PythonInterp REQUIRED)
 add_test(python_test ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/account/test.py)
 
 # export environment variables needed by the test
-set_tests_properties(python_test
-  PROPERTIES
-    ENVIRONMENT
-      PYTHONPATH=${CMAKE_CURRENT_SOURCE_DIR}:$ENV{PYTHONPATH}
-      ACCOUNT_LIBRARY_DIR=${CMAKE_CURRENT_BINARY_DIR}/lib
-      ACCOUNT_INCLUDE_DIR=${CMAKE_CURRENT_SOURCE_DIR}/account
+set_property(
+  TEST python_test
+  PROPERTY ENVIRONMENT
+    PYTHONPATH=${CMAKE_CURRENT_SOURCE_DIR}:$ENV{PYTHONPATH}
```

Comment:
What about using the `env:` stanza in `menu.yml` to preemptively set these env-vars on the Windows?

Reply:
I think this will be needed in chapter 9. Unless we wish to run ctest here too which is probably a good idea but then we need to refactor the custom.sh logic a bit to run both targets and custom.sh.